Mds/device sdk 508649 (#3513)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/customauthenticatoradminconsole.md

1. In the Admin Console, go to **Security** > **Authenticators**.
2. On the **Setup** tab, click **Add Authenticator**.

> 3. Click **Add** on the **Custom Authenticator** tile.

Author: I have to assume this is correct as I know nothing about it. It's a copy of the Android file.

4. Configure the following settings:

>    * **Authenticator name:** Type a name for the custom authenticator. This is the name that is used when you sign in using the authenticator.

Author: I don't know enough about the area to suggest a good name and I'd rather keep this as close to the Android text as possible. For now I'm going to leave it.

   * **Add to existing application:** Select the application that receives the push MFA prompt.
   * **User Verification:** Select an option to determine whether users must provide a PIN or biometric verification during authentication.
      * **Preferred** - User verification is optional.
      * **Required** - User verification is required.
   * **Authenticator logo:** Select the logo that users see on authentication screens.
      * **Browse files** - Click this button and select an SVG file from the file selection dialog.
      * **Use default logo** - Click the lock icon to use the default logo.
5. Configure the connection to the push notification service:

>    * **FCM configuration:** Select the connection to the FCM service that you want the custom authenticator to use.

Author: Good catch. I changed it. (there are stack selector files for the names, not sure why it wasn't used here... perhaps you can't use a stack selector file in a stack selector file?

6. Select the checkbox to agree to Okta's Terms and Conditions.

>    > **Note:** By adding this feature, you agree on behalf of the entity you represent that it is your sole responsibility to provide any required notices and disclosures to end users, including any necessary information from [https://www.okta.com/privacy-policy](https://www.okta.com/privacy-policy).

Reviewer: ```suggestion
   > **Note:** By adding this feature, you agree on behalf of the entity you represent that it's your sole responsibility to provide any required notices and disclosures to end users, including any necessary information from [https://www.okta.com/privacy-policy](https://www.okta.com/privacy-policy).
```

Author: Done
